apps/member/src/pages/SupportListPage/components/SupportTableRow/SupportAnswerSection.tsx

```diff
@@ -0,0 +1,97 @@
+import { useEffect, useState } from 'react';
+
+import { ROLE_LEVEL } from '@constants/state';
+import SupportAnswerInput from '@pages/SupportListPage/components/SupportAnswerInput';
+import { formattedDate } from '@utils/date';
+
+import { MemberProfileType } from '@type/member';
+import { SupportAnswerItem } from '@type/support';
+
+interface SupportAnswerSectionProps {
+  supportId: number;
+  answerData?: SupportAnswerItem;
+  myProfile: MemberProfileType;
+  isAnswered: boolean;
+}
+
+const SupportAnswerSection = ({
+  supportId,
+  answerData,
+  myProfile,
+  isAnswered,
+}: SupportAnswerSectionProps) => {
+  const [isAnswerEdit, setIsAnswerEdit] = useState(false);
+  const [answer, setAnswer] = useState<SupportAnswerItem>({});
+
+  const isAdmin = myProfile.roleLevel >= ROLE_LEVEL.ADMIN;
+
+  const handleAnswerEditToggle = () => setIsAnswerEdit((prev) => !prev);
+  const handleAnswerChange = (newAnswer: SupportAnswerItem) =>
+    setAnswer(newAnswer);
+
+  useEffect(() => {
+    if (isAnswerEdit && answerData) {
+      setAnswer({ ...answerData, id: supportId });
+    } else if (isAnswerEdit && !answerData) {
+      setAnswer({
+        id: supportId,
+        content: '',
+        responder: myProfile.name,
+        createdAt: new Date().toISOString(),
+      });
+    }
```

Comment:
`isAnswerEdit` 조건이 중복 사용되고 있어서, 다음과 같이 수정하면 좋을 것 같아요!
```suggestion
  if (!isAnswerEdit) return;

  if (answerData) {
    setAnswer({ ...answerData, id: supportId });
    return;
  }

  setAnswer({
    id: supportId,
    content: '',
    responder: myProfile.name,
    createdAt: new Date().toISOString(),
  });
```